src/App.test.tsx

```diff
@@ -1,11 +1,9 @@
 import React from 'react';
 import { render } from '@testing-library/react';
-import App from './App';
-import {Curiosity} from "./Curiosity";
+import {parseApiData} from "./Components/ApiFetch/ApodApiFetch";
 
-test('check app renders and page contains header text', () => {
-    let app = render(<App />);
-    const header = app.getByText('Mars Mission');
-    expect(header).toBeInTheDocument();
+
+test('check that function returns JSON', () => {
+    expect(parseApiData()).not.toBeNull();
```

Comment:
I'll come over and have a chat about testing API calls! :) 

a) We don't really want our app to _actually_ go an make an API call, because then our tests depend on external things like the internet connection.
b) I don't think this test actually tests what we want it to!